src/components/ui/AntDesign.jsx

```diff
@@ -0,0 +1,267 @@
+                    </Space>
+                </Col>
+                <Col span={12}>
+                    <QuestionCircleFilled style={{color: "grey"}} />
+                    <Text type="secondary">Click in list to book</Text>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <InfoCircleFilled style={{color: "grey"}} />
+                    <Text type="secondary">No Pending sign-ups to be booked</Text>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <Space direction="vertical">
+                        <Space align="center">
+                            <InfoCircleFilled/>
+                            <Title level={3}>Contact person</Title>
+                        </Space>
+                        <Space>
+                            <Avatar shape="square" size={50}>
+                                {props.data.person.first_name.charAt(0)}
+                                {props.data.person.last_name.charAt(0)}
+                            </Avatar>
+                            <Space direction="vertical">
+                                <Title level={4}>
+                                    {props.data.person.first_name} {props.data.person.last_name}
+                                </Title>
+                                <Text type="secondary">Drag another person here or select one to change</Text>
+                            </Space>
+                        </Space>
+                    </Space>
+                </Col>
+            </Row>
+            <Row gutter={[0, 40]}>
+                <Col span={24}>
+                    <Space direction="vertical">
+                        <Space align="center">
+                            <InfoCircleFilled/>
+                            <Title level={3}>Booked participants</Title>
+                        </Space>
+                        <Cascader
+                            action={props.data.action}
+                            onChange={onChange}
+                            placeholder="Please select"
+                            showSearch={{ filter }}
+                        />
+                    </Space>
+                </Col>
+            </Row>
+            </Col>
+            <Col flex="2rem"/>
+        </Row>
+)};
+
+const EditAction = props => {
+
+    const { Title, Text, Link } = Typography;
+    const { TextArea } = Input;
+
+    return(
+        <Row gutter={[0, 100]} wrap={false}>
+        <Col flex="2rem"/>
+        <Col span={24} flex="auto">
+            <Title level={2}>
+                Edit action
+            </Title>
+            <Space direction="vertical">
+                <Text>Campaign</Text>
+                <Cascader placeholder="Please select" />
+                <Text>Date</Text>
+                <DatePicker/>
+                <Text>Start time</Text>
+                <TimePicker/>
```

Comment:
I tried typing a time into this, just to see how that would work, but because it's state is not managed the value is ignored after I blur. I can't tell if this is because it is supposed to ignore typed input, or if it's because the state is not set.

Would you mind controlling this component? I.e. setting some state on change, and passing the state back to the component.

Reply:
It seems like it ignores all input if you don't press OK? Should I still control the component?

Reply:
I see! No need to test anything more then. I think we should make a note of this, because this is definitely a shortcoming of the library IMO.

Reply:
I'll make a note about that!